pique/functions.php

```diff
@@ -382,3 +382,24 @@ function pique_get_random_posts( $number = 1, $post_type = 'post' ) {
 	require get_template_directory() . '/inc/woocommerce.php';
 }
 
+/**
+ * Modifies Jetpack Testimonial Post Loop to include
+ * post ordering by menu_order
+ *
+ * New Order - Date - DESC, Menu Order - ASC
+ *
+ * @param array $query
+ * @return array $query
+ */
+function modify_jetpack_testimonial_archive_query( $query ) {
```

Comment:
As this change affects Jetpack, this function should be added to `inc/jetpack.php` instead of to `functions.php`.

Reply:
Moved to `inc/jetpack.php`

Reply:
You haven't pushed the changes to the repo yet, have you?

Reply:
Just committed changes